packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "salesforcedx-vscode-lightning",
-  "displayName": "Lightning Component Framework",
-  "description": "Provides syntax highlighting for Lightning components",
+  "displayName": "Aura Components",
```

Comment:
@lcampos - This doesn't have any impact on our analytics, right since we use the actual name itself?

Reply:
No impact, the name used for the analytics is defined in the `telemetry.ts` for each extension. Do we need to update that ?